apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -1698,9 +1698,20 @@ export default Vue.extend({
       if (selectedRowIndex === this.selectedRowIndex) return
 
       const position = (this.limit * (this.page - 1)) + (row.getPosition() || 0)
+      const cleanData = { ...this.$bks.cleanData(data, this.tableColumns) }
+
+      // un-stringify json fields that have been stringified by tabulator mutator
+      Object.keys(cleanData).forEach((key) => {
```

Comment:
@azmy60 you should use the LanguageData#getLanguageByContent to figure it out if it's a text field that's actually JSON.

Reply:
You're right.